gql/builder.go

```diff
@@ -402,6 +411,26 @@ func ResolveListField(name string, parent string) graphql.FieldResolveFn {
 			return nil, err
 		}
 
+		if qr, ok := p.Context.Value(QueryReporterContextKey).(QueryReporter); ok && qr != nil {
+			var lf ListFunctions
+			if sortParams != nil {
+				lf.SortField = sortParams.field
+				lf.SortOrder = sortParams.order
+			}
+
+			if filter != nil {
+				filterJSON, err := json.Marshal(filter.json)
```

Comment:
Doing a marshal in a reporting section is a bit less than ideal. What you really want is the name, operation and argument. All of those are available in newListFilter why not just put them in as string attributes in the type rather than doing all the JSON bits?